src/Configuration/Option.php

```diff
@@ -62,12 +62,6 @@ final class Option
      */
     public const IMPORT_SHORT_CLASSES = 'import_short_classes';
 
-    /**
-     * @deprecated Narrow too complex import settings to more practical use.
-     * @var string
-     */
-    public const IMPORT_DOC_BLOCKS = 'import_doc_blocks';
-
```

Comment:
This does currently crash our CI's. Is this expected to be removed in a patch version release? @TomasVotruba

Reply:
I'm sorry about that. This option is no longer supported, so you can remove it.
Rector is still in dev 0.x, where breaks can happen.

Reply:
Thx for the fast reponse. We will remove it from our configuration.